### ISSUE TITLE
Display world hopping limitations

### ISSUE THREAD

**Issue by Snitt** [runelite/runelite#6353]
**Is your feature request related to a problem? Please describe.**
Hopping from world to world too often causes Jagex to kick you offline and limit how often you can switch worlds. Not knowing this limit can be a pain as you can sometimes be left on the login screen and forced to re-type your password. If Jagex's server side logic for rate limiting world hops is known, it'd be nice to build this into the client to show when you can/can't world hop to minimise how many times you're left looking at your login screen.

**Describe the solution you'd like**
This massively depends on Jagex's rate limiting logic but generally i'd imagine something long the lines of showing the user how many "log ins" they have per period of time. Jagex's side might be more complicated though.

**Describe alternatives you've considered**
Simply put, if the logic isn't known or able to be figured out or determined (Maybe it's based on the amount of players online at one time?) then this feature simply can't be implemented nicely enough.

**Comment by forsco** [comment]
If it were known we would have implemented this.

**Comment by deathbeam** [comment]
Duplicate of #4786
